Assets/Scripts/MapPlayerTracker.cs

```diff
@@ -49,20 +52,40 @@ public void SelectNode(MapNode mapNode)
 
         private void SendPlayerToNode(MapNode mapNode)
         {
+            mapNode.ShowSwirlAnimation();
             Locked = lockAfterSelecting;
             mapManager.CurrentMap.path.Add(mapNode.Node.point);
             mapManager.SaveMap();
             view.SetAttainableNodes();
             view.SetLineColors();
             mapNode.ShowSwirlAnimation();
 
+
             DOTween.Sequence().AppendInterval(enterNodeDelay).OnComplete(() => EnterNode(mapNode));
+
+
+            if(mapNode.Node.nodeType == NodeType.MinorEnemy){
+               StartCoroutine(waiter("arenaScene"));
+            }
+        }
+
+        IEnumerator waiter(String arenaName){
+            yield return new WaitForSeconds((float)0.7);
+            OpenScene(arenaName);
+        }
+
+        public void OpenScene(String sceneName){
+            SceneManager.LoadScene(sceneName);
+        }
+
+        public void returnToTree(){
+            SceneManager.LoadScene("SampleSceneMap");
```

Comment:
The Back button is triggered when the game is paused, i.e. `timeScale = 0`.
As such, returning to the map leads to a paused map.

Simply add the following line directly below this one, and it should be fixed.
`Time.timeScale = 1.0f;`